src/vs/platform/terminal/common/terminal.ts

```diff
@@ -115,6 +115,7 @@ export const enum TerminalSettingId {
 	SmoothScrolling = 'terminal.integrated.smoothScrolling',
 	IgnoreBracketedPasteMode = 'terminal.integrated.ignoreBracketedPasteMode',
 	FocusAfterRun = 'terminal.integrated.focusAfterRun',
+	InitialHint = 'terminal.integrated.initialHint',
```

Comment:
This should move into initialHint contrib too, see https://github.com/microsoft/vscode/blob/0b028460cfd1a1c30978d2a61b2d35e4f70996c2/src/vs/workbench/contrib/terminalContrib/suggest/common/terminalSuggestConfiguration.ts#L15 for how I've been doing this after the refactor

Reply:
I saw that, but wondered if it was overkill to do an enum for just one setting. done